### PR TITLE
New export format

### PR DESCRIPTION
**Pull request by tovop**
# Summary

## Added
- Export time series to SIMA H5 file format.

## Fixed
- Previously errors were raised when having timeseries from different drives in the same time series database. This is now resolved.

### CODE REVIEW

**Comment by tovop** [comment]
When the time is right @eneelo, I guess you can help out with defining test cases for the mean load- and corrosion dependent S-N curve.

**Comment by tovop** [comment]
> When the time is right @eneelo, I guess you can help out with defining test cases for the mean load- and corrosion dependent S-N curve.

Disregard. Content moved to other branch and review will be dealt with in a different PR.

**Comment by EBGlom** [comment]
closes #89 , I persume @tovop

File: docs/source/examples/mooring_fatigue_mean_load.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+"""
```
Comment (EBGlom) [inline docs/source/examples/mooring_fatigue_mean_load.py:1]: Move this example to new-sn-model branch

File: docs/source/examples.rst [hunk 2]
```diff
@@ -96,11 +96,31 @@ Compare the cycle range and range-mean distribution from several time series usi
 Calculate fatigue damage in mooring lines
```
Comment (EBGlom) [inline docs/source/examples.rst:96]: rollback for this PR

File: qats/io/sima.py [hunk 3]
```diff
@@ -234,6 +235,39 @@ def read_ascii_data(path, ind=None, verbose=False):
...
+        Time (s)
+    data : array
+        Time series data
+    dt : float, optional
+        Time step on file, default 0.2 seconds
+    description : str, optional
+        Description printed to file
+    """
+    # resample to constant dt
+    duration = time[-1] - time[0]
```
[the remark this answers is not in the document]
Comment (EBGlom) [reply qats/io/sima.py:256]: I can add our fix, and then we can refine that into something more robust

[the remark this answers is not in the document]
Comment (EBGlom) [reply qats/io/sima.py]: @tovop , have a look

[the remark this answers is not in the document]
Comment (tovop) [reply qats/io/sima.py]: Super, thanks.

**Comment by tovop** [comment]
It now works with versions 3.7-3.9. Ready for review @EBGlom

File: README.md [hunk 4]
```diff
@@ -142,10 +142,10 @@ qats -h
...
 
-The automated tests are run using [Tox](https://tox.readthedocs.io/en/latest/).
+The automated tests are run using [unittest](https://docs.python.org/3/library/unittest.html/).
 
 ```console
-tox
+python -m unittest discover 
 ```
 
 The test automation is configured in the file `tox.ini`.
```
Comment (EBGlom) [inline README.md:151]: Remove

**Review by EBGlom — APPROVED** [review]
Approved, but one concern; testing on Py3.9 takes ~17 min with GitHub Actions

**Comment by tovop** [comment]
It was indeed an unfortunate combination of dependencies for Python 3.9. I have now relaxed the dependency specification to ease build and tests for any version in the Python test matrix.

**Pull request merged**